EventsExpress/ClientApp/src/components/login/login.js

```diff
@@ -2,15 +2,31 @@ import React, { Component } from 'react';
 import { Field, reduxForm } from 'redux-form';
 import DialogActions from '@material-ui/core/DialogActions';
 import Button from '@material-ui/core/Button';
-import Module from '../helpers';
 import GoogleLogin from '../../containers/GoogleLogin';
 import LoginFacebook from '../../containers/FacebookLogin';
 import TwitterLogin from '../../containers/TwitterLogin';
 import ErrorMessages from '../shared/errorMessage';
 import { connect } from 'react-redux';
 import { Redirect } from 'react-router';
+import { renderTextField } from '../helpers/form-helpers';
 
-const { validate, renderTextField } = Module;
+const validate = values => {
+    const errors = {}
+    const requiredFields = [
+        'email',
+        'password',
+    ]
+    requiredFields.forEach(field => {
+        if (!values[field]) {
+            errors[field] = 'Required'
+        }
+    })
+    if (values.email &&
```

Comment:
let move email validations into independent validator and reuse it in every place we need